docs/source/models.rst

```diff
@@ -3996,6 +3996,37 @@ set the `env_manager` argument when calling :py:func:`mlflow.pyfunc.spark_udf`.
     df = spark_df.withColumn("prediction", pyfunc_udf(struct("name", "age")))
 
 
+If you want to call `:py:func:`mlflow.pyfunc.spark_udf` through Databricks connect in remote client, you need to build the model environment in Databricks runtime first.
+
+.. rubric:: Example
+
+.. code-block:: python
+
+    from mlflow.pyfunc import build_model_env
+
+    # Build the model env and save it as an archive file to the provided UC volume directory
+    # and print the saved model env archive file path (like '/Volumes/.../.../XXXXX.tar.gz')
+    print(build_model_env(model_uri, "/Volumes/..."))
```

Comment:
```suggestion
    print(build_model_env(model_uri, "/Volumes/..."))
    
    # <comment>
    spark.conf.get("spark.databricks.clusterUsageTags.clusterId")
```

can we add `spark.conf.get("spark.databricks.clusterUsageTags.clusterId")` here to clarify where it should be executed?